### DISCUSSION TITLE
Bug: GitHub App > Settings > App Language

### DISCUSSION THREAD

**Discussion by c42573n** [community/community#52362] [Mobile]
There is no way to change the App Language.
When I click in the Settings of the App on App Language to change it, it opens the iPhone Settings App.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.107.0 (95566662)
Device: iPhone 12 Pro Max (iOS 16.4.1)
</details>

**Comment by Aheyachy** [comment]
Aheya Ahmed Chowdhury

**Comment by nolantrent** [comment]
> There is no way to change the App Language.
> When I click in the Settings of the App on App Language to change it, it opens the iPhone Settings App.
> 
> 
> 
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.107.0 (95566662)
> Device: iPhone 12 Pro Max (iOS 16.4.1)
> </details>

[](https://twitter.com/stakeusa/status/1645265086049845248?s=46)

**Reply by nolantrent** [reply]
@nolantrent

**Comment by markhaehnel** [comment]
I'm experiencing the exact same issue!

Github for iOS 1.139.1 (116337430)
iPhone 12 Pro Max (iOS 17.1.1)

**Comment by HB3YDM** [comment]
Is there a way to automatically translate the text within the app into my native other language for better understanding?

**Comment by philippfrank** [comment]
Why was this closed? I (German) work in a US company and I am used to the en-us GitHub UI. Still, I don't want to switch my Phone language.
